Lib/fontTools/feaLib/builder_test.py

```diff
@@ -117,11 +118,8 @@ def expect_ttx(self, font, expected_ttx):
             self.fail("TTX output is different from expected")
 
     def build(self, featureFile):
-        path = self.temp_path(suffix=".fea")
-        with open(path, "w", encoding="utf-8") as outfile:
-            outfile.write(featureFile)
         font = makeTTFont()
-        addOpenTypeFeatures(font, path)
+        addOpenTypeFeaturesFromString(font, featureFile)
```

Comment:
Perhaps it might be worth having a separate test for parsing/building from a filesystem path input?